src/potentiometer-element.ts

```diff
@@ -120,6 +147,26 @@ export class PotentiometerElement extends LitElement {
     `;
   }
 
+  observeInputInteractions() {
+    this.inputEl = this.shadowRoot?.querySelector('.hide-input');
+    const potentiometerKnob: SVGRectElement | null | undefined = this.shadowRoot?.querySelector(
+      '#rotating'
+    );
+
+    this.inputEl?.addEventListener('focus', () => {
```

Comment:
You can probably achieve the same with css, e.g. something like:

`input:focus + svg #rotating` or similar

Reply:
Thanks I actually tried input:focus + svg #rotating in one of my tries and it didn't work but maybe I selected it by class instead of id.. I'll check it out again :) 
regarding the lit-element handlers totally right I wasn't thinking :)